src/Platforms/SQLitePlatform.php

```diff
@@ -748,16 +744,12 @@ private function getSimpleAlterTableSQL(TableDiff $diff): array|false
         $sql = [];
 
         foreach ($diff->getAddedColumns() as $column) {
-            $definition = array_merge([
-                'unique' => null,
-                'autoincrement' => null,
-                'default' => null,
```

Comment:
All of these keys are either always returned by `$column->toArray()` (`autoincrement`, `default`) or are never returned (`unique`). Initializing them with `null` values doesn't affect any downstream code.

Even if `unique` is set on a column as a platform option (which isn't done anywhere), it's not handled anywhere. And even if it was, there is not code that would handle the `null` value of the `unique` key differently than the absence of this key (there's no code that handles this key at all).

If a custom implementation of `AbstractPlatform#getColumnDeclarationSQL()` relies on this key being always present, it can be fixed by adding `?? null` to the custom code. This is extremely unlikely and trivially fixable.